src/model/article-vor.v1.yaml

```diff
@@ -23,10 +23,44 @@ allOf:
             required:
               - doi
               - content
+        decisionLetter:
+            type: object
+            properties:
+                doi:
+                    $ref: ../misc/doi.yaml
+                description:
+                    type: array
+                    items:
+                        $ref: ../blocks/paragraph.v1.yaml
+                    minItems: 1
+                content:
+                    type: array
+                    items:
+                        $ref: ../misc/blocks-all.v1.yaml
+                    minItems: 1
+            required:
+              - doi
+              - description
+              - content
```

Comment:
`decisionLetter` and `authorResponse` should be next to each other. Had to reorder due to https://github.com/BigstickCarpet/json-schema-ref-parser/issues/24.